generate_secureblue_iso.sh

```diff
@@ -12,97 +13,76 @@ function is_yes {
     esac
 }
 
-desktop_options=("kinoite" "cinnamon" "bluefin" "silverblue" "sericea" "wayblue-wayfire" "wayblue-sway" "wayblue-river" "wayblue-hyprland")
-desktop_options_asus=("silverblue" "kinoite")
+# Define image configurations
+declare -A image_configs=(
+    ["server"]="Server"
+    ["server-zfs"]="Server"
+    ["silverblue"]="Silverblue"
```

Comment:
silverblue also has asus options

Reply:
Yes, indeed, I overlooked the `asus` option for Silverblue and Kinoite. It has been fixed now. Apologies for any delays in communication; we might be in different time zones, leading to asynchronous interactions.

Reply:
@HryshcIlya no worries! it's FOSS, there's no rush :smile: